src/pages/settings/Security/TwoFactorAuth/IsEnabledPage.js

```diff
@@ -0,0 +1,96 @@
+import React, {Component} from 'react';
+import {Text, View} from 'react-native';
+import HeaderWithCloseButton from '../../../../components/HeaderWithCloseButton';
+import Navigation from '../../../../libs/Navigation/Navigation';
+import ScreenWrapper from '../../../../components/ScreenWrapper';
+import withLocalize, {withLocalizePropTypes} from '../../../../components/withLocalize';
+import ROUTES from '../../../../ROUTES';
+import Section from '../../../../components/Section';
+import * as Illustrations from '../../../../components/Icon/Illustrations';
+import * as Expensicons from '../../../../components/Icon/Expensicons';
+import themeColors from '../../../../styles/themes/default';
+import styles from '../../../../styles/styles';
+import ConfirmModal from '../../../../components/ConfirmModal';
+
+const propTypes = {
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {};
+
+class IsEnabledPage extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            isConfirmModalVisible: false,
+        };
+
+        this.showConfirmModal = this.showConfirmModal.bind(this);
+        this.hideConfirmModal = this.hideConfirmModal.bind(this);
+        this.hideAndNavigate = this.hideAndNavigate.bind(this);
+    }
+
+    showConfirmModal() {
+        this.setState({isConfirmModalVisible: true});
+    }
+
+    hideConfirmModal() {
+        this.setState({isConfirmModalVisible: false});
+    }
+
+    hideAndNavigate() {
+        this.hideConfirmModal();
+        Navigation.navigate(ROUTES.SETTINGS_TWO_FACTOR_DISABLE);
+    }
+
+    render() {
+        return (
+            <ScreenWrapper>
+                <HeaderWithCloseButton
+                    title={this.props.translate('twoFactorAuth.headerTitle')}
+                    shouldShowBackButton
+                    onBackButtonPress={() => Navigation.navigate(ROUTES.SETTINGS_SECURITY)}
+                    onCloseButtonPress={() => Navigation.dismissModal(true)}
+                />
+
+                <Section
+                    title={this.props.translate('twoFactorAuth.twoFactorAuthEnabled')}
+                    icon={Illustrations.ShieldYellow}
+                    menuItems={[{
+                        title: this.props.translate('twoFactorAuth.disableTwoFactorAuth'),
+                        onPress: () => { this.showConfirmModal(); },
```

Comment:
I think you can call directly the `showConfirmModal` without the need of creating another function, like

```
onPress: this.showConfirmModal,
```